code/game/objects/items/weapons/shields.dm

```diff
@@ -155,10 +155,12 @@
 	var/undeploy_time = 1 SECONDS
 	///Whether it is wired
 	var/is_wired = FALSE
+	///Wether it can be deployed on the tad/alamo
+	var/restricted_deployment = TRUE
```

Comment:
why do we need a var?

Reply:
Saw it was like that with the others vars, so I figured it was needed for this var as well. Went ahead and removed it.